### ISSUE TITLE
Help message with Quarkus CLI mentions a different executable

### ISSUE THREAD

**Issue by gastaldi** [quarkusio/quarkus#15644]
**Describe the bug**
Running `qs --help` (installed through `jbang app install --name qs cli@quarkusio`) shows that the executable name is `quarkus` in the _usage_ section

```bash
/ ᐅ qs --help
[jbang] Resolving dependencies...
[jbang]     Resolving io.quarkus:quarkus-cli:1.11.3.Final:runner...Done
[jbang] Dependencies resolved
Usage: quarkus [-ehV] [--verbose] [COMMAND]
  -e, --errors    Produce execution error messages.
  -h, --help      Show this help message and exit.
  -V, --version   Print version information and exit.
      --verbose   Verbose mode.
Commands:
  build         Build your quarkus project
  clean         Clean current project
  create        Create a new quarkus project.
  create-jbang  Create a new quarkus jbang project.
  list          List installed (default) or installable extensions.
  add           Add extension(s) to current project.
  remove, rm    Remove an extension from this project.
  dev           Execute project in live coding dev mode
```

**Expected behavior**
It should display `qs`

**Actual behavior**
It shows `quarkus` as the executable name

**To Reproduce**

1. Install JBang: `sdk install jbang` 
2. Install Quarkus Cli: `jbang app install --name qs cli@quarkusio`
3. Run  `qs --help`

**Comment by abid1998** [comment]
I can start working on this. I am a new contributor so if anyone can point me into the right direction it would be great !

**Comment by gastaldi** [comment]
I think it's a matter of changing the command name in https://github.com/quarkusio/quarkus/blob/e33f4d7a9aca2da2440e6536fc7d207e8f0d8dc0/devtools/cli/src/main/java/io/quarkus/cli/QuarkusCli.java#L21

and probably removing the `aliases` field, since they would be the same

**Comment by abid1998** [comment]
Lol looks like it was fixed :D

**Comment by aadityasinha-dotcom** [comment]
Can I work on this issue?

**Comment by gastaldi** [comment]
Sorry, but this was already fixed in #15747. Closed
